docs/topics/con_http-api-design-tradeoffs.adoc

```diff
@@ -2,12 +2,12 @@
 
 = {name-example-http-api} design tradeoffs
 
-.Design Tradeoffs
+.Design tradeoffs
 [width="100%",options="header"]
 |====================================================================
 |Pros           |Cons
 a|
-* The booster enables fast prototyping.
+* The example enables fast prototyping.
```

Comment:
In this instance, "example application" makes more sense than "example". Consider rephrasing as: The example application enables fast prototyping.

Reply:
Good comment, thank you.